### PR TITLE
Updating package per CRAN reviewer commnets.

### PR DESCRIPTION
**Pull request by arko-bhattacharya**

### CODE REVIEW

File: R/internal.R [hunk 1]
```diff
@@ -100,7 +100,7 @@ scidb_unpack_to_dataframe = function(db, query, ...)
                               ssl_verifypeer=0))
   resp = curl_fetch_memory(uri, h)
   if (resp$status_code > 299) stop("HTTP error", resp$status_code)
-  if (DEBUG) message("Data transfer time ", (proc.time() - dt2)[3])
+  if (DEBUG) message("Data transfer time ", round((proc.time() - dt2)[3]), 4)
```
Comment (ksens) [inline R/internal.R:103]: There is a typo here -- `, 4` should go inside the `round` function.

File: R/internal.R [hunk 2]
```diff
@@ -914,7 +914,7 @@ matvec2scidb = function(db, X,
   ans = gsub("\n", "", gsub("\r", "", ans))
   if (DEBUG)
   {
-    message("Data upload time ", (proc.time() - td1)[3], "\n")
+    message("Data upload time ", round((proc.time() - td1)[3], 4), "\n")
```
Comment (ksens) [inline R/internal.R:917]: `message` does not need a `\n` at the end... `cat` needs it..

Best if you remove such occurrences -- unless you really needed two lines.

File: R/utility.R [hunk 3]
```diff
@@ -519,10 +519,10 @@ scidb_prefix = function(db, expression=NULL)
 getpwd = function(prompt="Password:")
 {
   if (grepl("mingw", R.version["os"])) return(readline(sprintf("%s ", prompt)))
-  cat(prompt, " ")
```
Comment (ksens) [inline R/utility.R:522]: I take back what I said earlier -- probably better to use `cat` in `getpwd`. We can defend the choice of `cat` to CRAN reviewer as needed.

**Review by ksens — CHANGES_REQUESTED** [review]
@arko-bhattacharya 

Suggested a few minor changes.

File: R/internal.R (hunk 1 above)
Comment (arko-bhattacharya) [reply R/internal.R:103]: Done

File: R/internal.R (hunk 2 above)
Comment (arko-bhattacharya) [reply R/internal.R:917]: Done

File: R/utility.R (hunk 3 above)
Comment (arko-bhattacharya) [reply R/utility.R:522]: Done

**Pull request merged**